homeassistant/components/zwave_js/entity.py

```diff
@@ -42,11 +42,6 @@ async def async_added_to_hass(self) -> None:
         self.async_on_remove(
             self.info.node.on(EVENT_VALUE_UPDATED, self._value_changed)
         )
-        self.async_on_remove(
-            async_dispatcher_connect(
-                self.hass, f"{DOMAIN}_update_{self.info.value_id}", self._value_changed
```

Comment:
@marcelveldt I couldn't find this signal. Please confirm that we can remove this.

Reply:
This can be removed as discovery already handles it now